mappings/net/minecraft/block/RenderLayer.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_1921 net/minecraft/block/RenderLayer
+	FIELD field_20805 name Ljava/lang/String;
+	FIELD field_20806 expectedBufferSize I
+	FIELD field_20807 beginAction Ljava/lang/Runnable;
+	FIELD field_20808 endAction Ljava/lang/Runnable;
+	FIELD field_20972 vertexFormat Lnet/minecraft/class_293;
+	FIELD field_20973 drawMode I
+	FIELD field_9175 CUTOUT_MIPPED Lnet/minecraft/class_1921;
```

Comment:
Did stitch fail to infer this name?

Reply:
No. Not me who mapped it though, this was already mapped in `BlockRenderLayer`. I unmapped it.